.gitignore

```diff
@@ -4,3 +4,6 @@ target
 .DS_Store
 
 yarn.lock
+
+/.bsp
+sbt
```

Comment:
What's this `sbt` directory? I don't think I've ever seen it.

Reply:
oh, it just the script from https://github.com/paulp/sbt-extras .Will remove